woocommerce/wc-admin
# Add COD method to default payment gateway recommendations

Fixes (part of) #6855 

* Updates some naming around "method" -> "RecommendedPaymentGateway"
* Hooks up the payment gateway list to the data store
* Adds the COD method to fallback/default recommended gateways.
* Adds some optimization around the loops used to parse gateways.

### Screenshots
<img width="706" alt="Screen Shot 2021-05-24 at 6 04 48 PM" src="https://user-images.githubusercontent.com/10561050/119412843-8bc7d400-bcba-11eb-85f3-d9dd629363b3.png">


### Detailed test instructions:

1. Deactivate the remote tester plugin and delete any remote payment transients.
2. Disable the COD method in WC Settings if enabled.
3. Visit the payments task.
4. Enable the payment method.
5. Note that enabling it moves it to the enabled list.
6. Note that the "Manage" button appears and works as expected.
